reportgen/analysis_modules/performance_plots.py

```diff
@@ -131,48 +141,24 @@ def run(self, api) -> List[BaseOutput]:
                     }
                     response = FrontEndCall(api, endpoint='scores').post(request)
 
-                    fpr = response['data']['roc_curve']['fpr']
-                    tpr = response['data']['roc_curve']['tpr']
-                    thresholds = response['data']['roc_curve']['thresholds']
-                    res = np.abs(np.array(thresholds) - binary_threshold)
-                    threshold_indx = np.argmin(res)
+                    if response['kind'] == 'NORMAL':
+                        fpr = response['data']['roc_curve']['fpr']
+                        tpr = response['data']['roc_curve']['tpr']
+                        thresholds = response['data']['roc_curve']['thresholds']
+                        res = np.abs(np.array(thresholds) - binary_threshold)
+                        threshold_indx = np.argmin(res)
+
+                        metrics[model_id][dataset][source['name']]['fpr'] = fpr
+                        metrics[model_id][dataset][source['name']]['tpr'] = tpr
+                        metrics[model_id][dataset][source['name']]['threshold_indx'] = threshold_indx
 
-                    metrics[model_id][dataset][source['name']]['fpr'] = fpr
-                    metrics[model_id][dataset][source['name']]['tpr'] = tpr
-                    metrics[model_id][dataset][source['name']]['threshold_indx'] = threshold_indx
+                    else:
+                        output_modules += [DescriptiveTextBlock(f"Performance scores are not available "
```

Comment:
Can we return any more information?  Are there other response['kind']s besides NORMAL?

Reply:
Added a warning which prints the error message received from the FE.